ui/admin/app/components/header-nav/index.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class HeaderNavComponent extends Component {
+  // =services
+
+  @service session;
+  @service scope;
+  @service router;
```

Comment:
Is this service used by the component?

Reply:
You are right, this component does not need the service router, I delete it.